src/test/test-execve.c

```diff
@@ -0,0 +1,24 @@
+#include "exec-util.h"
```

Comment:
SPDX line is missing

Reply:
Should we write a test that checks that every file starts with an SPDX line?

Reply:
...but then we'll need a test that checks the test :-O
but yeah, that would be nice, although it will need some heuristics

Reply:
Duh.

At least we don't use the copyright and license headers anymore.